src/System.Data.SqlClient/tests/Tools/TDS/TDS.EndPoint/Cryptography/CryptKeyProviderInfo.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.SqlServer.TDS.EndPoint.Cryptography
+{
+    /// <summary>
+    /// Structure that contains provider information
+    /// </summary>
+    [StructLayout(LayoutKind.Sequential)]
+    internal struct CryptKeyProviderInfo
```

Comment:
Where is all of this interop stuff used?  Do we not have .NET APIs that provides this functionality?  What's the plan for this on Unix where these don't exist?